app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md

```diff
@@ -124,26 +122,17 @@ Choose referees who could comment on things like your:
 
 Your referee will also be asked if they know of any reason why you should not work with children.
 
-Ideal referees could include:
-
-* your university tutor or supervisor
-* your current line manager at work
-* your previous employer
-* a teacher at a school where you work or volunteer
-* a supplier or client you’ve worked with (if you’re self-employed)
-
-Referees should not be family members, partners or friends.
+You'll need different referees depending on your circumstances. If you:
 
-It’s important to have at least one academic or professional reference.
+* are still studying or graduated in the last 5 years, you'll need an academic reference
+* already work in a school, you'll need a reference from your headteacher
+* are applying for a salaried course, you'll need a reference from an employer
 
 Training providers will accept a character reference, such as a mentor or someone you know from volunteering, as a second reference.
 
-If you’re applying for a salaried course, one of your references must be from
-an employer.
-
-You can request as many references as you like to increase the chances of getting 2 quickly.
+Referees should not be family members, partners or friends.
 
-Once you've received 2 or more references, you can select the 2 you want to include in your application.
+You'll provide your referee details when you apply, but they will not be contacted until you accept a place on a course. You can then amend your referee details if necessary.
 
 If you’re struggling to choose your referees, [get free one-on-one support from a
```

Comment:
I know this isn't a change on this PR but do we normally say "one-on-one" support when we talk about advisers?

Reply:
think it's normally 'one-to-one' - will change